NEMO/templates/customizations.html

```diff
@@ -231,7 +231,7 @@ <h3 id="safety_intro">Introduction for &quot;<a href="{% url 'safety' %}" target
 		<div class="panel panel-default">
 			<div class="panel-body">
 				<h3 id="facility_rules_tuto">&quot;<a href="{% url 'facility_rules' %}" target="_blank">{{ facility_name }} rules tutorial</a>&quot; page</h3>
-				<p>The {{ facility_name }} rules tutorial is an opportunity to provide new users with a tutorial to your lab operating procedures and rules. The HTML you upload is rendered with the Django template engine. You can use JavaScript (including jQuery) within the page. Completion of the HTML form should be POSTed to the 'facility_rules' URL. Upon completion, the user's "training required" attribute is set to false, and they are able to make reservations and control tools.</p>
+				<p>The {{ facility_name }} rules tutorial is an opportunity to provide new users with a tutorial to your lab operating procedures and rules. The HTML you upload is rendered with the Django template engine. You can use JavaScript (including jQuery) within the page. The form should include {% templatetag openblock %} csrf_token {% templatetag closeblock %} to specify which user should receive credit for completing the tutorial.  Completion of the HTML form should be POSTed to the '/facility_rules_tutorial/' URL. Upon completion, the user's "training required" attribute is set to false, and they are able to make reservations and control tools.</p>
```

Comment:
I would remove the "to specify which user should receive credit for completing the tutorial" part. The way I read it is that the csrf token had a link to the user somehow. no big deal
also would you mind changing the post to {% url 'facility_rules' %} that way if we decide to change the actual url it will be resolved to the right one.

Reply:
Will do.

Reply:
I am about to release NEMO 3.1.0 so if you want to change it quick I'll wait a bit

Reply:
OK.  I did another commit to that branch so the pull request should be updated.